src/System.Runtime.Numerics/tests/BigInteger/ctor.cs

```diff
@@ -924,7 +924,7 @@ public static void RunCtorByteArrayTests()
             // ctor(byte[]): array is large
             for (int i = 0; i < s_samples; i++)
             {
-                tempByteArray = new byte[s_random.Next(16384, 2097152)];
+                tempByteArray = new byte[s_random.Next(16384, 262144)];
```

Comment:
The default stack size [seems](http://stackoverflow.com/questions/823724) to be only 1 MB. I'll change that to `fixed` -- and I'll update multiplication too!